RO-2890: Øke zoomnivå for kartutsnitt ved ny observasjon

amish1188 opened the pull request:
Starter man ny observasjon fra hovedkart, skal kartet i set-location vise samme utsnitt som i hoved kart.
Hvis man starter på /registration/new uten hovedkart, skal den vise Norge. Usikker hvorfor jeg ikke får vist Svalbard selv om jeg bruker samme koordinater nå på unknownMapCenter + samme zoom nivå 5 som viser på hovedkartet, men kanskje det ikke er så viktig?

amish1188 committed:
sette samme zoom nivå som i hoved kart ved ny observasjon

amish1188 committed:
set norway as defualt when starting new observation without main map

jorgkv reviewed (changes requested):
Hvis man starter en ny observasjon med lat lon spesifisert, via feks https://victorious-water-056410803-805.westeurope.azurestaticapps.net/registration/new/70?lat=60.374757&lon=8.179665 
, så er det nå vanskelig å se hvor man er, og man må zoome seg manuelt langt inn. Vi kunne enten lagt til en parameter for zoomnivå som iskart.no kan bruke, eller bare bruke zoomnivå 5 hvis man ikke sender med lat lon under oppstarten. Hva tror du?

amish1188 committed:
oppdater iskart zoom nivå

jorgkv reviewed (changes requested):
Klarer vi å beholde samme zoomnivå for sett skredposisjon og sett flomområde som vi hadde tidligere?

amish1188 commented:
> Klarer vi å beholde samme zoomnivå for sett skredposisjon og sett flomområde som vi hadde tidligere?

jeg tror du trenger kanskje å forklare det til meg når du er tilbake på jobb ;)

jorgkv committed:
Fiks feil med cluster i set-obs-loc-map

jorgkv committed:
Endre til hvitt ikon for å skille fra tidligere lokasjoner

jorgkv committed:
Ikke bruk async ngOnInit

jorgkv committed:
La parent komponent styre min zoom nivå + bugfiksing

- Blant annet: Nå er det mulig å endre lokasjon selv
  om man tidligere hadde valgt et gammelt sted
- Custom navn fjernes ved klikk på gammelt sted

jorgkv commented:
Jeg gjorde en del endringer på denne. Nå kan parent komponent til set-location-in-map selv styre min zoom nivå, da er det enklere å tillate langt utzoomet noen steder, men ikke i sett flomområde / sett skred start-stopp feks.

Jeg fiksa også en del småfeil jeg fant mens jeg satt med dette, blant annet:
- Cluster for tidligere steder i kartet vises
- Kan nå velge et annet sted selv om det tidligere var valgt et gammelt sted
- Endret / custom navn fjernes ved klikk på gammelt sted.
- Noe opprydning (flere linjer fjerna her enn nye..). Blant annet knytta til det å lytte på kartutsnitt som nå bare håndteres via mapservice.
